app/src/main/res/layout/topic_play_story_summary.xml

```diff
@@ -24,15 +27,15 @@
         android:layout_height="match_parent"
         android:indeterminate="false"
         android:max="100"
-        android:progress="55"
+        android:progress="@{storyProgressPercentage}"
         android:progressDrawable="@drawable/circular_progress"
         android:secondaryProgress="100" />
       <TextView
         android:id="@+id/story_progress_percentage"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:gravity="center"
-        android:text="55%"
+        android:text="@{String.format(@string/topic_story_progress_percentage, storyProgressPercentage)}"
```

Comment:
Doesn't data binding have a syntax to simplify this rather than using String.format? E.g. I thought you could do something like ``@string/some_string(value)``.

Reply:
See: https://stackoverflow.com/a/38984088/3689782.

Reply:
Updated attribute to `@{@string/topic_story_progress_percentage(storyProgressPercentage)}`